visual-tests/series/comparison.js

```diff
@@ -1,88 +0,0 @@
-(function(d3, fc) {
-    'use strict';
-
-    var data = [
-        fc.data.random.financial().startDate(new Date(2014, 1, 1))(50),
-        fc.data.random.financial().startDate(new Date(2013, 12, 15))(50)
-    ];
-
-    var percentageChange = fc.indicator.algorithm.calculator.percentageChange()
```

Comment:
In that case it sounds like we should just stick it in util for now then, I'm reluctant to remove it completely.